README.md

```diff
@@ -5,29 +5,22 @@
 
 Convert JSON schema to Google BigQuery schema
 
+This includes the ability to:
+
+(1) Create tables
+(2) Patch tables
+
+Further enhancements are planned: delete tables (dev only), create dataset, set data ACLs
+
+Note that some features involve bespoke interpretation of schema details suited to our environment.
+
 ## Install
 
     npm install jsonschema-bigquery
 
 ## Consume
 
-    const jsonSchemaBigquery = require('jsonschema-bigquery')
```

Comment:
Can you keep these code blocks as i see the modules primary function is to allow json schema to converted programmatically. The JSBQ command is an added extra.